src/lava/lib/dl/slayer/auto.py

```diff
@@ -25,7 +25,7 @@ def get_classes(neuron_type=None):
         neuron class and block class.
 
     """
-    if neuron_type is None or neuron_type == 'cuba' or neuron_type == 'LOIHI':
+    if neuron_type is None or neuron_type == 'CUBA' or neuron_type == 'LOIHI':
```

Comment:
Same as above, is there a reason to support both type names for the same thing?

Reply:
Yes for legacy reason.